### PR TITLE
Feature/howler audio

### PR DESCRIPTION
**Pull request by agmcleod**
#157

### CODE REVIEW

File: src/audio/audio.js [hunk 1]
```diff
@@ -265,10 +256,13 @@
 				}
 				sync_loading = true;
 			}
-
+			var urls = [];
+			for(var i = 0; i < this.audioFormats.length; i++) {
+				urls.push(sound.src + sound.name + "." + this.audioFormats[i] + me.loader.nocache);
+			}
```
Comment (parasyte) [inline src/audio/audio.js:262]: I like the `Array.prototype.map` function, better. ;)

Comment (agmcleod) [inline src/audio/audio.js:262]: I'm old fashioned i guess ;). Map is a pretty function, i use it in ruby quite a bit. I also know it tends to be a tad slower than the for counter party.

Comment (parasyte) [inline src/audio/audio.js:262]: For an upper bound of 5 array elements? ;)

Comment (agmcleod) [inline src/audio/audio.js:262]: Hello world has never run so slowly.

To be honest, I didn't even think of it that much. I just kinda wrote the array to build it haha.

**Pull request merged**